README.md

```diff
@@ -1,9 +1,81 @@
-# versionize_derive
+**Versionize is a framework for version tolerant serializion/deserialization of
+Rust data structures, designed for usecases that need fast deserialization 
+times and minimal size overhead. It does not aim to be a generic serialization 
+framework and only the [bincode](https://crates.io/crates/bincode) backend is 
+supported.**
+
+#### You may be looking for:
+- [Versionize documentation](https://docs.rs/versionize/)
+- [Releases](https://github.com/firecracker-microvm/versionize_derive/releases)
 
 ## Important note
 
-This crate is currently used for cross-version serialization with the [Firecracker snapshot-restore dev preview](https://github.com/firecracker-microvm/firecracker/tree/v0.23.0), but has not been tested for other use cases. It should be considered **experimental software** outside the Firecracker context. It’s likely that this crate will see both interface and implementation changes in the future.
+This crate is currently used for cross-version serialization with the 
+[Firecracker snapshot-restore dev preview](https://github.com/firecracker-microvm/firecracker/tree/v0.24.0), 
```

Comment:
This line still breaks the rule. Here is a [quick guide](https://www.rubycoloredglasses.com/2017/11/markdown-links-80-character-line-length/) on how to fix it.